test/views/email/PasscodeErrorViewSpec.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.email
+
+import views.html.email.PasscodeErrorView
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import play.twirl.api.Html
+import views.ViewBaseSpec
+
+class PasscodeErrorViewSpec extends ViewBaseSpec {
+
+  val injectedView: PasscodeErrorView = inject[PasscodeErrorView]
+  lazy val view: Html = injectedView()(user, messages, mockConfig)
+
+  lazy implicit val document: Document = Jsoup.parse(view.body)
+
+  "The passcode error view" should {
+
+
+    "have the correct document title" in {
+      document.title shouldBe "You need to start again - Business tax account - GOV.UK"
+    }
+
+    "have the correct heading" in {
+      document.getElementsByClass("heading-large").text() shouldBe "You need to start again"
+    }
+
+    "have the correct text for the first paragraph" in {
+      elementText("#content > article > p:nth-child(2)") shouldBe "This is because you have entered the wrong code too many times."
+    }
+
+    "have the correct text for the second paragraph" in {
+      elementText("#content > article > p:nth-child(3)") shouldBe "Return to your VAT account" + " to start the process again."
+    }
+
+    "have a link that takes you to VAT account" which {
```

Comment:
Could you add an extra test inside this brace to check the link href is the correct value in `mockConfig`. You can grab this with 

`element("#content > article > p:nth-child(3) > a").attr("href")`